src/src/com/microsoft/aad/adal/AuthenticationContext.java

```diff
@@ -241,7 +239,7 @@ public boolean contains(String key) {
                 }
 
                 @Override
-                public Iterator<TokenCacheItem> getAll() {
+                public Iterable<TokenCacheItem> getAll() {
```

Comment:
I meant you don't need this method at all.. `ITokenCacheStore` should be `Iterable< TokenCacheItem >`